### ISSUE TITLE
Version 2.0.0-alpha002 - NuGet "DirectoryNotFoundException" for "staticwebassets"

### ISSUE THREAD

**Issue by hfloyd** [limbo-works/Limbo.Umbraco.ModelsBuilder#15]
Just did the NuGet update to the latest version (2.0.0-alpha002). 

Build worked fine, but on running the site in a browser I get this: 

![image](https://user-images.githubusercontent.com/1252593/188010031-5cde4c93-ee09-4b36-814f-08dbc5108fd0.png)

```
An error occurred while starting the application.
DirectoryNotFoundException: C:\Users\Heather\.nuget\packages\limbo.umbraco.modelsbuilder\2.0.0-alpha002\staticwebassets\
Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(string root, ExclusionFilters filters)

DirectoryNotFoundException: C:\Users\Heather\.nuget\packages\limbo.umbraco.modelsbuilder\2.0.0-alpha002\staticwebassets\
Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(string root, ExclusionFilters filters)
Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(string root)
Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader+<>c.<UseStaticWebAssetsCore>b__1_0(string contentRoot)
Microsoft.AspNetCore.StaticWebAssets.ManifestStaticWebAssetFileProvider..ctor(StaticWebAssetManifest manifest, Func<string, IFileProvider> fileProviderFactory)
Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.UseStaticWebAssetsCore(IWebHostEnvironment environment, Stream manifest)
Microsoft.AspNetCore.Hosting.StaticWebAssets.StaticWebAssetsLoader.UseStaticWebAssets(IWebHostEnvironment environment, IConfiguration configuration)
Microsoft.AspNetCore.WebHost+<>c.<ConfigureWebDefaults>b__9_0(WebHostBuilderContext ctx, IConfigurationBuilder cb)
Microsoft.AspNetCore.Hosting.GenericWebHostBuilder+<>c__DisplayClass9_0.<ConfigureAppConfiguration>b__0(HostBuilderContext context, IConfigurationBuilder builder)
Microsoft.Extensions.Hosting.HostBuilder.BuildAppConfiguration()
Microsoft.Extensions.Hosting.HostBuilder.Build()
Umbraco.Cms.Web.Common.Hosting.UmbracoHostBuilderDecorator.Build()
UmbracoProject.Program.Main(string[] args) in Program.cs
+
            => CreateHostBuilder(args)

```

Any ideas?

**Comment by abjerner** [comment]
That is weird. I don't really have any good ideas for why the directory isn't there.

What is located inside your `C:\Users\Heather\.nuget\packages\limbo.umbraco.modelsbuilder\2.0.0-alpha002` directory? here is mine:

![image](https://user-images.githubusercontent.com/3634580/188012190-a6ce9290-65f3-442f-8cc3-2a67b93280a0.png)

The `staticwebassets` directory is also part of the package when exploring here:

https://nuget.info/packages/Limbo.Umbraco.ModelsBuilder/2.0.0-alpha002

How have you installed the package? I guess you already have tried this, but does it works after rebuilding in VS or running `dotnet restore` and `dotnet build`?

**Comment by hfloyd** [comment]
What's weird is that the 'staticwebassets' folder is there:
![image](https://user-images.githubusercontent.com/1252593/188013107-86122483-6fe4-4667-a42b-547175a1634a.png)

And these are the contents:
![image](https://user-images.githubusercontent.com/1252593/188013630-f473f01d-d34e-4018-9b28-5b945fd6209c.png)

(There are files inside those folders, so it seems all is present...)

I updated it via VisualStudio, and rebuilt. `dotnet restore` just shows `All projects are up-to-date for restore.`

**Comment by hfloyd** [comment]
I did some more general web searching on the issue and came across other software packaged with similar issues reported. 

One of the suggestions was to make sure that `iis_iusrs` has full control of the .nuget folder.... so I updated the permissions via Windows Explorer:

![image](https://user-images.githubusercontent.com/1252593/188015385-f6efdf02-d93e-4ad5-8701-dee9a5e42939.png)

After rebuilding in VS, the site loaded.

It seems to be a strange sort of requirement... but whatever... 🤷‍♂️

Thanks for your reply!
